ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/ipv4-acl-in-missing-on-l3-port-channel-wan-interface.yml

```diff
@@ -23,5 +23,4 @@ wan_path_groups:
     name: INTERNET
 
 expected_error_message: >-
-  'ipv4_acl_in' must be set on WAN interfaces where 'wan_carrier' is set,
-  unless the carrier is configured as 'trusted' under 'wan_carriers'. 'ipv4_acl_in' is missing on L3 Port-Channel 'Port-Channel5'.
+  A node variable 'cv_pathfinder_site' must be defined when 'wan_role' is 'client' and 'wan_mode' is 'cv-pathfinder'.
```

Comment:
We should not change negative tests drastically like this. It is no longer testing what we wanted it to test. The code execution is reordered when you change from `cached_property` to `structured_config_contributor` so it triggers another error instead. Please resolve the new error by setting the cv_pathfinder_site under the node so we get back to the original error.

Same for the other negative test.

Reply:
Apologies! Fixed it.